docs/source/using/image-caching.adoc

```diff
@@ -125,19 +125,19 @@ We recommend using this feature with caution.
 [[implicit-image-caching]]
 == Implicit Image Caching
 
-Image caching can be configured to automatically occur during the creation of the {project} VM while executing `minishift start`.
-To enable this feature you need to enable the `image-caching` property in the persistent configuration using the xref:../command-ref/minishift_config_set#[`minishift config set`] command:
+Image caching has been configured to automatically occur during the creation of the {project} VM while executing `minishift start`.
+After the completion of `minishift start`, caching occurs in a background process, the first time you use the xref:../command-ref/minishift_start#[`minishift start`] command.
```

Comment:
`After the completion of `minishift start`, caching occurs` => `After the completion of `minishift start`, image caching occurs`